crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -59,7 +59,7 @@ AEAD - Authenticated Encryption with Associated Data
 +-----------------------+-------------------------------+
 
 .. note::
-   APIs currently in mbedtls_extra, as well as APIs with no current HW TLS support, will be supported in upcoming releases of the nrf_cc3xx_mbedcrypto library.
+   APIs in mbedtls_extra and APIs with no Mbed TLS support will be supported in the future.
```

Comment:
Statement untrue with the establishment and standardization on PSA Crypto. We should remove this sentence altogether.